site2/docs/reference-pulsar-admin.md

```diff
@@ -2267,35 +2230,28 @@ Options
 |`-l`, `--ledgerId`|The ledger id |0|
 |`-e`, `--entryId`|The entry id |0|
 
-### `get-backlog-quotas`
-Get the backlog quota policies for a topic.
+### `enable-deduplication`
+Enable a deduplication policy for a topic.
```

Comment:
In order to make topic-level and namespace-level consistent, these commands have been deprecated. See CmdTopics#CmdTopics for details.
The deprecated commands are in CmdTopics#initDeprecatedCommands